userguide/content/en/docs/get-started/docsy-as-module/installation-prerequisites.md

```diff
@@ -82,8 +82,8 @@ If the `go` language is not installed on your system yet or if you need to upgra
 
 Hugo's commands for module management require that the `git` client is installed on your system. Check whether `git` is already present in your system:
 
-```bash
-git version
+```console
+$ git version
 git version 2.38.1.windows.1
```

Comment:
Can you bring this to latest version 2.39.1?
Also, maybe we can omit `.windows.1` to keep it platform neutral?

Reply:
Done ✨